occurrences/signals.py

```diff
@@ -43,3 +78,40 @@ def remove_message_id(sender, message, **kwargs):
     # ESP in order to avoid the message being identified as spam by some strict
     # filters.
     message.extra_headers.pop("Message-ID", None)
+
+
+@receiver([m2m_changed], sender=Occurrence.languages.through)
```

Comment:
Not sure if this receiver also covers the case when an occurrence is removed? What would happen then

Reply:
Need to check that! Well thought. I could even add an unit test for it.

Reply:
Added a new signal receiver and a test for that one.